tests/functional/adapter/snapshot_tests/files.py

```diff
@@ -0,0 +1,68 @@
+SEED_CSV = """
```

Comment:
`fixtures.py` might be a better name here.

Reply:
This file contains contents that are inserted into fixtures, but aren't themselves fixtures. When looking at the corresponding tests in `dbt-core`, the name `files.py` was used for the same concept.

Reply:
I split `files.py` into `seedds.py` and `snapshots.py` to be consistent with how other adapter repos have handled this. That left me with a very small model that I just put in the test file itself instead of creating a separate file for it.

Reply:
missed this convo on my review

Reply:
Is everyone comfortable with these file names then?